.github/workflows/CompatHelper.yml

```diff
@@ -1,16 +1,16 @@
-name: CompatHelper
+name: "CompatHelper"
+
 on:
   schedule:
     - cron: 0 0 * * *
   workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+
 jobs:
   CompatHelper:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Pkg.add("CompatHelper")
-        run: julia -e 'using Pkg; Pkg.add("CompatHelper")'
-      - name: CompatHelper.main()
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          COMPATHELPER_PRIV: ${{ secrets.DOCUMENTER_KEY }}
-        run: julia -e 'using CompatHelper; CompatHelper.main()'
+    name: "CompatHelper"
```

Comment:
```suggestion
```
I don't think you need to explicitly name this job, since it automatically inherits that name from the definition on line 12 (if I'm not mistaken). Doesn't hurt either though!

Reply:
I changed the job to `compat-helper` and kept this line, following the conventions of other workflows here and in the docs of https://github.com/ITensor/ITensorActions.